index.bs

```diff
@@ -140,6 +145,38 @@ The <dfn attribute for="Document">prerendering</dfn> getter steps are to return
 The <dfn attribute for="Document">onprerenderingchange</dfn> attribute is an [=event handler IDL attribute=] corresponding to the <dfn event for="Document">prerenderingchange</dfn> [=event handler event type=].
 
 
+<h2 id="link-rel-prerender">Link type "<dfn attr-value for="link/rel"><code>prerender</code></dfn>"</h2>
+
+<em>The following section would be added as a new sub-section of [[HTML]]'s <a href="https://html.spec.whatwg.org/multipage/links.html#linkTypes">Link types</a> section. This should replace the prerender documentation in the Resource Hints standard</em>
+
+The <{link/rel/prerender}> keyword may be used with <{link}> elements. This keyword creates an <a spec=HTML>external resource link</a>. This keyword is <a spec=HTML>body-ok</a>.
+
+The appropriate times to <a spec=HTML>fetch and process the linked resource</a> for such a link are:
```

Comment:
I think some of these (I only checked "browsing-context connected") are exported, so you can use `[=browsing-context connected=]` instead of `<a spec="HTML">browsing context connected</a>`. The `spec=HTML` is a hack to get access to non-exported terms without having to put them in the links block at the top.

Reply:
Nice, Done.